defaults/main.yml

```diff
@@ -67,6 +67,9 @@ gitlab_runner_restart_state: restarted
 # default value for log_format
 # gitlab_runner_log_format: runner
 
+# default value for force accept self signed certificates
+force_accept_gitlab_server_self_signed
```

Comment:
This is not valid YAML, is it? The colon and value are missing.

This causes the following error:

```
ERROR! We were unable to read either as JSON nor YAML, these are the errors we got from each:
JSON: Expecting value: line 1 column 1 (char 0)
Syntax Error while loading YAML.
  could not find expected ':'
The error appears to be in '/root/.ansible/roles/riemers.gitlab-runner/defaults/main.yml': line 73, column 1, but may
be elsewhere in the file depending on the exact syntax problem.
The offending line appears to be:
# controls diffs for assemle config file
^ here
```

Reply:
Sorry, did not notice that. Should have added
```yaml
force_accept_gitlab_server_self_signed: false
```

rather than
```yaml
force_accept_gitlab_server_self_signed
```

I'll open another PR (provided that the problem is still in master branch) with a fix

Reply:
Ah, just saw your PR related to this. Thanks!